bundles/org.eclipse.passage.loc.dashboard.ui/src/org/eclipse/passage/loc/dashboard/ui/panel/DefaultDashboardPanelAdvisor.java

```diff
@@ -282,25 +282,42 @@ public void updateLicenseInfo(LicenseRegistry licenseRegistry) {
 
 	@Override
 	public void createFooterInfo(Composite parent) {
-		Label summary = new Label(parent, SWT.NONE);
 		GridDataFactory gdf = GridDataFactory.fillDefaults().align(SWT.CENTER, SWT.TOP).indent(0, 10).grab(true, false);
+		appendSummary(parent, gdf);
+		appendIssueLicenseButton(parent, gdf, //
+				DashboardUiMessages.DefaultDashboardPanelAdvisor_btn_issue_text, //
+				DashboardUiMessages.DefaultDashboardPanelAdvisor_btn_issue_description, //
+				this::executeIssueLicenseCommand);
+		appendIssueLicenseButton(parent, gdf, //
+				DashboardUiMessages.DefaultDashboardPanelAdvisor_btn_issue_floating_text, //
+				DashboardUiMessages.DefaultDashboardPanelAdvisor_btn_issue_floating_description, //
+				this::executeIssueFloatingLicenseCommand);
+	}
+
+	private void appendSummary(Composite parent, GridDataFactory gdf) {
```

Comment:
The idea was to have this panel fully overridable from downstream products, may be we should make it protected